telemetry/definitions/commonDefinitions.json

```diff
@@ -3530,6 +3540,26 @@
                 }
             ]
         },
+        {
```

Comment:
We need to add an event to trigger for when projects have not yet initiated transformation to get user details. This is a P0 requirement.